Userland/Applications/SpaceAnalyzer/main.cpp

```diff
@@ -76,25 +81,24 @@ struct MountInfo {
     DeprecatedString source;
 };
 
-static void fill_mounts(Vector<MountInfo>& output)
+static ErrorOr<void> fill_mounts(Vector<MountInfo>& output)
 {
     // Output info about currently mounted filesystems.
-    auto file = Core::File::construct("/sys/kernel/df");
-    if (!file->open(Core::OpenMode::ReadOnly)) {
-        warnln("Failed to open {}: {}", file->name(), file->error_string());
-        return;
-    }
+    auto file = TRY(Core::Stream::File::open_file_or_standard_stream("/sys/kernel/df"sv, Core::Stream::OpenMode::Read));
```

Comment:
We are opening from a static file name here, so we don't need the `_or_standard_stream` part. `Core::Stream::File::open()` should be more than enough for this.